### PR TITLE
Remove duplicate bbr-sdk-release resource

### PR DESCRIPTION
**Pull request by taylorsilva**
Saw no reason to have the same resource declaration twice 🤷‍♂

### CODE REVIEW

**Comment by jomsie** [comment]
@taylorsilva looks like branch is out of date, i'll let you update!

File: pipelines/concourse.yml [hunk 1]
```diff
@@ -1500,12 +1500,6 @@ resources:
   source:
     repository: pivotal-cf/credhub-release
 
-- name: backup-and-restore-sdk-release
```
Comment (jomsie) [inline pipelines/concourse.yml]: just for my own reference, the bbr-sdk-release resource def is on line 1485

**Comment by taylorsilva** [comment]
@jomsie rebased. Review again please! 😁

**Comment by jomsie** [comment]
looks good to me!!

**Pull request merged**